packages/@aws-cdk/cdk/lib/secret.ts

```diff
@@ -1,94 +1,47 @@
-import { CfnParameter } from './cfn-parameter';
-import { Construct } from './construct';
-import { Token } from './token';
+import { Token } from "./token";
+import { unresolved } from "./unresolved";
 
 /**
- * A token that represents a value that's expected to be a secret, like
- * passwords and keys.
+ * Work with secret values in the CDK
  *
- * It is recommended to use the `SecretParameter` construct in order to import
- * secret values from the SSM Parameter Store instead of storing them in your
- * code.
+ * Secret values in the CDK (such as those retrieved from SecretsManager) are
+ * represented as regular strings, just like other values that are only
+ * available at deployment time.
  *
- * However, you can also just pass in values, like any other token: `new Secret('bla')`
+ * To help you avoid accidental mistakes which would lead to you putting your
+ * secret values directly into a CloudFormation template, constructs that take
+ * secret values will not allow you to pass in a literal secret value. They do
+ * so by calling `Secret.assertSafeSecret()`.
+ *
+ * You can escape the check by calling `Secret.unsafeSecret()`, but doing
```

Comment:
Maybe instead of `Secret.unsafeSecret` we can do `Secret.clearText` or `Secret.plainText`? Less about policy, more about mechanism...